src/workos.ts

```diff
@@ -1,11 +1,16 @@
 import { WorkOS } from '@workos-inc/node';
 import { WORKOS_API_HOSTNAME, WORKOS_API_HTTPS, WORKOS_API_KEY, WORKOS_API_PORT } from './env-variables.js';
+import { version } from '../package.json';
```

Comment:
I'm fairly certain this will break in production, as our tsconfig doesn't know how to handle json files.

Reply:
I'm dumb and didn't see that you added `resolveJsonModule`.

Reply:
well, the json part is fine but it actually does break production 😂 😭 . it changes the compiled output such that all the ts files get nested under `src` and breaks things. going to push up a new iteration.